osc-server/src/main/java/org/osc/core/broker/util/db/upgrade/ReleaseUpgradeMgr.java

```diff
@@ -239,6 +239,8 @@ private static void performUpdateChain(int curDbVer, Statement stmt, EncryptionA
                 upgrade84to85(stmt);
             case 85:
                 upgrade85to86(stmt);
+            case 86:
+                upgrade86to87(stmt);
```

Comment:
I think we are clashing here. I also have 87.

Reply:
Whoever does the merge after needs to handle the conflict as usual.